src/plotting/README.md

```diff
@@ -0,0 +1,57 @@
+Scripts for plotting data contained in either the sqlite or mysql databases.
+
+plot_barcode_family.py and plot_line_family.py are built to use sqlite as the backend. 
+
+Scripts which contain "mysql" in title reference the mysql database.
+
+
+# Usage
+
+Each script is designed to be called independently and produce a single plot. Calls are made to be to python with flags as necessary as defined below.
+
+Example call:
+```sh
+python3 plot_mysql_dir_sensor_sat.py --sidb ../satinfo 
+```
+
+## Flag options 
+
+--sidb provides the location of the satinfo folder for plots which go down to the satellite level to include the black/white list 
+if not included, then the plot will be all black and not sure any blue lines 
```

Comment:
Looking at that rich text version, this line does not break from the one above, hindering its readability. Also, what is meant by "...not sure any blue lines"?

Reply:
it was supposed to be show and not sure, mistype, will update good catch